test/http-invocation.test.js

```diff
@@ -193,6 +193,146 @@ describe('HttpInvocation', function() {
       inv.transformResponse(res, body, cb);
     }
   });
+
+  describe('createRequest', function() {
+    it('should create a simple request', function() {
+      var inv = givenInvocationForEndpoint(null, []);
+
```

Comment:
Please remove extra line here and
group together `var` declarations, for example:
```javascript
 var a = 'Hello';
 var b = 'How are you';

 function printMessage(a, b){
   console.log(a + ', ' + b);
}

```
 
Same applies to other it blocks